keras_cv/layers/preprocessing3d/global_z_rotation.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.ops.point_cloud import coordinate_transform
+from keras_cv.ops.point_cloud import wrap_angle_radians
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+
+
+class GlobalZRotation(base_augmentation_layer_3d.BaseAugmentationLayer3D):
```

Comment:
What do you all think about calling this RandomZRotation (to follow our general convention for KPL naming)

Reply:
Sure, I can change it to GlobalRandomZRotation to distinguish it with other non-global object level data augmentations.

Reply:
Just git pushed another version. Please let me know if you have other suggestions.